compiler/scenario-service/protos/scenario_service.proto

```diff
@@ -104,6 +104,11 @@ message UpdateContextRequest {
   UpdateModules update_modules = 2;
   UpdatePackages update_packages = 3;
   bool noValidation = 4; // if true, does not run the package validations
+  // The lf v1 minor of the modules in `load_packages`
+  // The empty sting means no change of version.
+  // Can be changed from one call to the other only if all the previously
```

Comment:
Being able to change this seems like a fair amount of complexity for no gain afaict. Why not specify the LF version as a CLI parameter to the scenario service? Afaict, we can fix that statically in the compiler and we definitely don’t change it during runtime.

Reply:
redesigned.